Source/SwiftyCamViewController.swift

```diff
@@ -917,9 +932,19 @@ open class SwiftyCamViewController: UIViewController {
 
 	/// Enable or disable flash for photo
 
-	fileprivate func changeFlashSettings(device: AVCaptureDevice, mode: AVCaptureDevice.FlashMode) {
+fileprivate func changeFlashSettings(device: AVCaptureDevice, mode: FlashMode) {
 		do {
 			try device.lockForConfiguration()
+            let mode:AVCaptureDevice.FlashMode
+            switch flashMode{
```

Comment:
Would look a little cleaner if we move this method to the enum so we can just call `mode.AVFlashMode`